scripts/aggregate-changelogs/script.py

```diff
@@ -191,9 +191,9 @@ def get_cluster_releases(repo_shortname):
 
 def normalize_version(v):
     """
-    Removes a 'v' prefix from the version string
+    Removes a 'v' prefix from the version string when it is followd by a number
     """
-    if v.startswith('v'):
+    if bool(re.match(r'^v\d', v)):
```

Comment:
Solves this issue -> https://github.com/giantswarm/docs/pull/2312/files#diff-c5c3179b31f3a20c1fa1d1a349d8c19395c706a65e34c1ba24b5ec5a0ca5abd1R10